pandas/core/indexes/datetimes.py

```diff
@@ -1706,9 +1706,67 @@ def freq(self, value):
                                   "The microseconds of the datetime")
     nanosecond = _field_accessor('nanosecond', 'ns',
                                  "The nanoseconds of the datetime")
-    weekofyear = _field_accessor('weekofyear', 'woy',
-                                 "The week ordinal of the year")
+    _weekofyear_doc = """
+    The week ordinal of the year.
+
+    Return the week ordinal of the year. Note that there can be
+    counter intuitive edge cases around a year change. For example
+    the first of january could be in week 52 of the previous year.
+    Monday indicates the start of a new week.
+
+    Returns
+    -------
+    ndarray of integers indicating the week number
```

Comment:
I thikn

    days : ndarray
        integers indicating the week number

ndarray is the type, the rest is the ddescription.

Reply:
Just to confirm, what you're suggesting is different than the docstring guide for the sprint. 

```
Returns
-------
int
    The sum of `num1` and `num2`
```

example found here: https://python-sprints.github.io/pandas/guide/pandas_docstring.html